### PR TITLE
build: confluent example added. Close #2

### LINKED ISSUE #2: Add Confluent example
We want to add a confluent example, ideally one that uses the `schema registry`. 

A possible integration could be with https://github.com/marcosschroh/python-schema-registry-client

### PR DESCRIPTION
**Pull request by marcosschroh**

### CODE REVIEW

File: examples/confluent-example/confluent_example/serializers.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from typing import Dict
+import aiokafka
+
+from schema_registry.serializers import AsyncAvroMessageSerializer
+
+
+class AvroSerializer(AsyncAvroMessageSerializer):
```
Comment (Woile) [inline examples/confluent-example/confluent_example/serializers.py:7]: I'm starting to wonder if some of this serializers should come with kstream itself

File: examples/confluent-example/README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+## Confluent example
```
Comment (Woile) [inline examples/confluent-example/README.md:1]: Maybe check the titles in this section.

**Review by Woile — APPROVED** [review]

File: examples/confluent-example/confluent_example/serializers.py (hunk 1 above)
Comment (marcosschroh) [reply examples/confluent-example/confluent_example/serializers.py:7]: Good question!  Maybe for now we can include in `kstreams` `serializers` that can be created with `native python modules` like `json` and `pickle`. If end users demands other `serializers` like `avro`, we can add them.

Comment (Woile) [reply examples/confluent-example/confluent_example/serializers.py:7]: For now we are fine I think, but we have to keep an eye if a pattern emerges from the examples

**Pull request merged**